cmd/lotus-shed/chain_index.go

```diff
@@ -14,37 +13,24 @@ import (
 	lcli "github.com/filecoin-project/lotus/cli"
 )
 
-func withCategory(cat string, cmd *cli.Command) *cli.Command {
-	cmd.Category = strings.ToUpper(cat)
-	return cmd
-}
-
 var chainIndexCmds = &cli.Command{
-	Name:            "chainindex",
-	Usage:           "Commands related to managing the chainindex",
-	HideHelpCommand: true,
+	Name:  "chainindex",
+	Usage: "Commands related to managing the chainindex",
 	Subcommands: []*cli.Command{
-		withCategory("chainindex", validateBackfillChainIndexCmd),
+		validateBackfillChainIndexCmd,
 	},
 }
 
 var validateBackfillChainIndexCmd = &cli.Command{
 	Name:  "validate-backfill",
 	Usage: "Validates and optionally backfills the chainindex for a range of epochs",
 	Description: `
-lotus-shed chainindex validate-backfill --from <start_epoch> --to <end_epoch> [--backfill] [--log-good]
+lotus-shed chainindex validate-backfill --from <start_epoch> --to <end_epoch> [--backfill] [--log-good] [--quiet]
 
 The command validates the chain index entries for each epoch in the specified range, checking for missing or
 inconsistent entries (i.e. the indexed data does not match the actual chain state). If '--backfill' is enabled
 (which it is by default), it will attempt to backfill any missing entries using the 'ChainValidateIndex' API.
 
-Parameters:
```

Comment:
removed this because it duplicates what urfave/cli/v2 prints out below this text